src/views/available.js

```diff
@@ -43,7 +44,7 @@ export const Available = () => {
                     setApps(r.data);
                 })
                 .catch(e => {
-                    setApps({})
+                    setApps({"filebrowser":{"name":"File Browser","app_id":"filebrowser","description":"File Browser - a utility for browsing files through a web interface","detail":"File Browser provides a web interface for browsing files in a cloud environment.","docs":"https://filebrowser.org/","spec":"https://github.com/helxplatform/app-support-prototype/raw/master/dockstore-yaml-proposals/filebrowser/docker-compose.yaml","minimum_resources":{"cpus":"1","gpus":0,"memory":"4000M"},"maximum_resources":{"cpus":"8","gpus":0,"memory":"64000M"}},"jupyter-ds":{"name":"Jupyter Data Science","app_id":"jupyter-ds","description":"Jupyter DataScience - A Jupyter notebook for exploring and visualizing data.","detail":"Includes R, Julia, and Python.","docs":"https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-datascience-notebook","spec":"https://github.com/helxplatform/app-support-prototype/raw/master/dockstore-yaml-proposals/jupyter-ds/docker-compose.yaml","minimum_resources":{"cpus":"1","gpus":0,"memory":"4000M"},"maximum_resources":{"cpus":"8","gpus":0,"memory":"64000M"}},"octave":{"name":"Octave","app_id":"octave","description":"A scientific programming language largely compatible with MATLAB.","detail":"GNU Octave is a high-level language, primarily intended for numerical computations.","docs":"https://www.gnu.org/software/octave","spec":"https://github.com/helxplatform/app-support-prototype/raw/master/dockstore-yaml-proposals/octave/docker-compose.yaml","minimum_resources":{"cpus":"1","gpus":0,"memory":"4000M"},"maximum_resources":{"cpus":"8","gpus":0,"memory":"64000M"}}})
```

Comment:
Is this is here to prevent the list from being empty? If so a couple thoughts:

- If the server gives an error what's the chances it won't error when somebody tries to launch the app from this list?
- What is the plan for maintaining this list relative to the common app list?

Reply:
No.. sorry these are just for error testing, I will remove them. I plan to add some notifications here to display errors, and the app listing components will says 'no apps available'.